app/models/hyrax/file_set.rb

```diff
@@ -24,5 +24,54 @@ def pcdm_object?
     def file_set?
       true
     end
+
+    ##
+    # Gives file metadata for the file filling the http://pcdm.org/OriginalFile use
+    # @return [FileMetadata] the FileMetadata resource of the original file
+    def original_file
+      Hyrax.query_service.custom_queries.find_many_file_metadata_by_use(resource: self, use: Hyrax::FileSet.original_file_use).first
+    end
+
+    ##
+    # Gives file metadata for the file filling the http://pcdm.org/ExtractedText use
+    # @return [FileMetadata] the FileMetadata resource of the extracted text
+    def extracted_text
+      Hyrax.query_service.custom_queries.find_many_file_metadata_by_use(resource: self, use: Hyrax::FileSet.extracted_text_use).first
+    end
+
+    ##
+    # Gives file metadata for the file filling the http://pcdm.org/Thumbnail use
+    # @return [FileMetadata] the FileMetadata resource of the thumbnail
+    def thumbnail
+      Hyrax.query_service.custom_queries.find_many_file_metadata_by_use(resource: self, use: Hyrax::FileSet.thumbnail_use).first
```

Comment:
i'm interested in refactoring to reduce duplication, but :+1: to moving forward with this as written.

Reply:
i.e., i think rebasing #4215 would resolve this, and moving forward with this will facilitate that refactor

Reply:
Done